src/engine/source/server/src/endpoints/eventEndpoint.cpp

```diff
@@ -213,6 +215,19 @@ EventEndpoint::EventEndpoint(
         [this, dumpFileHandler, isFloodedFileEnabled](const DatagramSocketEvent& eventSocket, DatagramSocketHandle& handle)
         {
             auto strRequest = std::string {eventSocket.data.get(), eventSocket.length};
+
+            // Received bytes per seconds
+            Metrics::instance().addCounterValue("Server.ReceivedBytesPerSeconds", static_cast<uint64_t>(eventSocket.length));
+
+            // Received events per seconds
+            Metrics::instance().addCounterValue("Server.ReceivedEventsPerSeconds", 1UL);
+
+            // Total events received
+            Metrics::instance().addCounterValue("Server.TotalEventsReceived", 1UL);
+
+            // Total bytes received
+            Metrics::instance().addCounterValue("Server.TotalBytesReceived", static_cast<uint64_t>(eventSocket.length));
+
```

Comment:
Quick fix, typo: Seconds must be "second". i.e.   Events received per **second**.